### DISCUSSION TITLE
[Blazor] ProtectedBrowserStorage become long after 25k

### DISCUSSION THREAD

**Discussion by tossnet** [dotnet/aspnetcore#55060] [Q&A]
I'm coming back to a small problem that was closed but not resolved (or not answered).

[https://github.com/dotnet/aspnetcore/issues/47945](https://github.com/dotnet/aspnetcore/issues/47945)

I've just created a new Blazor Server .NET 8 project and the problem is still there. As soon as you go to 24446 characters, the program seems to disconnect for a few seconds.

https://github.com/dotnet/aspnetcore/assets/3845786/38085791-785b-4d11-8b0d-af62bd547d19

**Comment by Alessio92** [comment]
Same here
it also seems like you can store the data as you want but you can only read 24445 bytes at a time, this is the test I did to prove it:

![image](https://github.com/dotnet/aspnetcore/assets/7070916/52b1a6f0-3303-4e65-8477-a4a4c55db68a)

**Comment by IrregularProgramming** [comment]
I have the same issue with SessionStorage, extremely frustrating. Any information on a workaround?
